packages/components/src/DraggableItemList.tsx

```diff
@@ -380,6 +384,7 @@ class DraggableItemList<T> extends PureComponent<
               rowHeight={rowHeight}
               selectedRanges={selectedRanges}
             />
+            {hasPlaceholder === true && provided.placeholder}
```

Comment:
If you make the `hasPlaceholder` type required, then this can just be `hasPlaceholder && provided.placeholder`